### DISCUSSION TITLE
How to add these lists to televizo

### DISCUSSION THREAD

**Discussion by asdkjasdfla** [iptv-org/community#595] [Q&A] [answered]
I tried adding to televizo program guide but it says link not correct while it's actually a link to the m3u in this repository

**Comment by Mrpickle30** [comment]
Add to the VLC PLAYER

**Comment by ghost** [comment, marked as answer]
works great for me

![Screenshot_20240904-220532_Televizo](https://github.com/user-attachments/assets/177a3f6e-65e1-44d4-9187-43237cf4e916)
 Also for epg

![Screenshot_20240904-220957_Televizo](https://github.com/user-attachments/assets/523c6666-f087-45c0-a7bd-9f056f0ad04c)
![Screenshot_20240904-221119_Televizo](https://github.com/user-attachments/assets/45e87f84-bc80-402b-8247-6fca484dadac)
